### PR TITLE
[core] Fix permissions in `no-response` workflow

### PR DESCRIPTION
**Pull request by cherniavskii**
Fixes a regression from https://github.com/mui/mui-x/pull/6396#discussion_r1008046273

### CODE REVIEW

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -15,6 +15,7 @@ jobs:
     permissions:
       contents: read
       pull-requests: write
+      issues: write
```
Comment (alexfauquette) [inline .github/workflows/no-response.yml:18]:
```suggestion
      issues: write
```

**Review by alexfauquette — COMMENTED** [review]
Not sure it needs write permission on PR

Did you try it on your repo to verify it fixes the problem?

File: .github/workflows/no-response.yml (hunk 1 above)
Comment (cherniavskii) [reply .github/workflows/no-response.yml:18]: Indeed, we only use this label for issues.

> Did you try it on your repo to verify it fixes the problem?

I have GitHub Actions disabled on my fork, but let me try if I can test it

Comment (cherniavskii) [reply .github/workflows/no-response.yml]: Seems to work on my fork: https://github.com/cherniavskii/material-ui-x/issues/4

**Review by oliviertassinari — APPROVED** [review]
Awesome, I have made a dumb mistake on this one, thanks for the 🧹 behind me. We can see the error in https://github.com/mui/mui-x/actions/runs/3350385087/jobs/5551157153

Screenshot 2022-10-29 at 15 45 50

---

I have applied the same fix to the other repositories.

---

I went on the older issues to fill the gap where the GitHub Action didn't run these last few days.

---

I have updated MUI org permissions to be:

Screenshot 2022-10-29 at 16 24 53


---

I have opened https://github.com/lee-dohm/no-response/pull/377 hoping that it might get released, our GitHub Actions should have failed.

**Review by alexfauquette — APPROVED** [review]

**Pull request merged**